<launch>
<arg name="model" default="$(find bioloid_c_description)/urdf/bioloid-halfarm.urdf"/>

<param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />

  <!-- send fake joint values -->
  <node name="joint_state_publisher" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui">
    <param name="use_gui" value="true"/>
  </node>

  <!-- Combine joint values -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>

  <!-- Show in Rviz   -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find bioloid_c_description)/rviz/visualize.rviz"/>

</launch>
